python/sglang/srt/configs/model_config.py

```diff
@@ -230,7 +230,7 @@ def _verify_quantization(self) -> None:
         # Parse quantization method from the HF model config, if available.
         quant_cfg = self._parse_quant_hf_config()
 
-        if quant_cfg is not None:
+        if quant_cfg is not None and self.quantization != "awq_turbomind":
```

Comment:
This part can likely be abstracted into a function, as there may be other quantization methods to check for in the future, such as `gptq_turbomind` and others.